internal/credential/vault/repository_credentials.go

```diff
@@ -0,0 +1,263 @@
+package vault
```

Comment:
Yes, this was intentional since this file has the "issue credentials" method. I don't think `repository_credential.go` is much better. If you can think of a better name let me know.